runtime/cere/src/lib.rs

```diff
@@ -1480,7 +1442,30 @@ pub type SignedExtra = (
 pub struct StakingMigrationV11OldPallet;
 impl Get<&'static str> for StakingMigrationV11OldPallet {
 	fn get() -> &'static str {
-		"VoterList"
+		"BagsList"
+	}
+}
+
+pub struct MigrateStakingPalletToV8;
+impl OnRuntimeUpgrade for MigrateStakingPalletToV8 {
+	fn on_runtime_upgrade() -> Weight {
+		pallet_staking::migrations::v8::migrate::<Runtime>()
```

Comment:
Inside the `v8` migration I see the following call: 
```
	pub(crate) fn unsafe_clear() {
		#[allow(deprecated)]
		crate::ListBags::<T, I>::remove_all(None);
		#[allow(deprecated)]
		crate::ListNodes::<T, I>::remove_all();
	}
```

As far as I understand, it will clear the list with all actual nominators from `voterList.listNodes` item and insert old ones from the `staking.nominators` item. I think, we should not execute this on Mainnet as it will cause the most recent nominators (that are not in `staking.nominators`) to be dropped.

Reply:
You are right. I found that we don't need to migrate to v8 since the migration to v8 is already executed. We can keep the code as is since the migration will be skipped [here](https://github.com/paritytech/substrate/blob/master/frame/staking/src/migrations.rs#L391)